department-of-veterans-affairs/caseflow
# ClayS/APPEALS-23402

Resolves [APPEALS-23403](https://jira.devops.va.gov/browse/JIRA-23402)

# Description
Updates informational text around POA for HLRs and SCs for VHA

## Acceptance Criteria
- [ ] For VHA users accessing Higher-Level Reviews and Supplemental Claims Decision Review screens: 
    - [ ] If claimant has an associated POA in VBMS, the following data and text will be displayed:
        - [ ] \<Attorney> \<Address> \<Email Address>
        - [ ] **Informational text: Power of Attorney (POA) data comes from VBMS.**
    - [ ] If claimant has no associated POA in VBMS, the following text will be displayed:
        - [ ] No POA data displays.
        - [ ] **Informational text: No known POA.** 


## Testing Plan

1. Go to [Jira Issue/Test Plan Link](https://jira.devops.va.gov/browse/JIRA-12345) or list them below
2. Log in as ACBAUERVVHAH
3. Navigate to the VHA decision review queue
4. Intake an HLR using 111111113 as the file number (this is a veteran without a POA)
5. Examine the HLR you just created and any other one. The new text should appear in the POA component
6. Repeat the process for supplemental claims


# Frontend
## User Facing Changes

<img width="1222" alt="image" src="https://github.com/department-of-veterans-affairs/caseflow/assets/17840614/25e5da90-d39c-4dae-a406-75b09be7997f">
<img width="1091" alt="image" src="https://github.com/department-of-veterans-affairs/caseflow/assets/17840614/9eb4c25a-1e98-41cd-bc6e-8c13a73a1711">


## Storybook Story
*For Frontend (Presentation) Components*

# Backend
## Database Changes
*Only for Schema Changes*


## Integrations: Adding endpoints for external APIs

# Best practices
## Code Documentation Updates

## Tests
### Test Coverage
Did you include any test coverage for your code? Check below:

### Code Climate
Your code does not add any new code climate offenses? If so why?

## Monitoring, Logging, Auditing, Error, and Exception Handling Checklist

### Monitoring

### Logging

### Auditing

### Error Handling

### Exception Handling
